assets/scss/themes/theme-dark/colors/_colors.scss

```diff
@@ -0,0 +1,52 @@
+/// Colors
+/// All colors theme darck
+
+$dp-color-white: #ffffff;
```

Comment:
maybe need to use the small one like `#fff`

Reply:
Prettier doesn't like it and changes it